app/serializers/atmosphere/virtual_machine_flavor_serializer.rb

```diff
@@ -5,7 +5,19 @@ class VirtualMachineFlavorSerializer < ActiveModel::Serializer
     attributes :id, :flavor_name,
       :cpu, :memory, :hdd, :hourly_cost,
       :compute_site_id, :id_at_site,
-      :supported_architectures, :active
+      :supported_architectures, :active, :cost_map
+
+    private
+
+    # Deprecated: returns max hourly cost
+    def hourly_cost
+      object.virtual_machine_flavor_os_families.max_by(&:hourly_cost).hourly_cost
+    end
```

Comment:
You don't need this method here, since it is already defined in `VirtualMachineFlavor`